frontend/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Stats from './components/stats';
+import Search from './components/search';
+
+class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      loading: true,
+      user: [],
+      name: null,
+      placeholder: null,
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      placeholder: event.target.value
+    })
+  }
+
+  handleSubmit = (event) => {
+    this.setState({
+      name: this.state.placeholder
+    })
+    event.preventDefault()
+  }
+
+  componentDidMount(){
```

Comment:
Now realizing that this componentDidMount could be commented out/removed and it would make more sense.